### DISCUSSION TITLE
Ladies & Gentlemen Meet! Aditya Vaidhya 🙏....

### DISCUSSION THREAD

**Discussion by TheCodeLord-INX** [community/community#72586] [GitHub Education]
### Hi everyone,

My name is Aditya Vaidhya, I am a Fresher at Indian Institute Of Technology, Madras pursuing my Bachelor's in Data Science. I love extracting useful insights from the Universe of cluttered Data & creating and making various Products that can contribute to the Better Future Of Our World.

**Comment by grandEL-dsouza** [comment]
Welcome to the world of data science.

**Reply by TheCodeLord-INX** [reply]
Thankyou so much! Sir.
